### PR TITLE
Feature/risk assessment grading

### PR DESCRIPTION
**Pull request by 9sneha-n**
### :pushpin: References

-   **Issue:** Closes https://app.clickup.com/t/8694tp1fw

### :memo: Implementation
1. Display risk grading table
2. Risk summary form
3. Risk Questionnaire form
4. Risk Grading form + calculation

### CODE REVIEW

File: src/data/repositories/OptionsD2Repository.ts [hunk 1]
```diff
@@ -6,12 +6,34 @@ import { OptionsRepository } from "../../domain/repositories/OptionsRepository";
...
+    MediumWeightedOption,
+    RiskAssessmentGrading,
+} from "../../domain/entities/risk-assessment/RiskAssessmentGrading";
+import { Future } from "../../domain/entities/generic/Future";
+
 const MAIN_SYNDROME_OPTION_SET_CODE = "AGENTS";
 const SUSPECTED_DISEASE_OPTION_SET_CODE = "RTSL_ZEB_OS_DISEASE";
 const NOTIFICATION_SOURCE_OPTION_SET_CODE = "RTSL_ZEB_OS_SOURCE";
 
 export class OptionsD2Repository implements OptionsRepository {
```
Comment (anagperal) [inline src/data/repositories/OptionsD2Repository.ts:32]: We'll need to refactor the options repository as soon as possible and create an entity for each option (with its corresponding repo) and join the options in the corresponding hook in the presentation layer.

File: src/webapp/pages/event-tracker/EventTrackerPage.tsx [hunk 2]
```diff
@@ -51,14 +68,38 @@ export const EventTrackerPage: React.FC = React.memo(() => {
...
+                        >
+                            {i18n.t("Create Risk Assessment")}
+                        </Button>
+                    ) : (
+                        <Button
+                            variant="outlined"
+                            color="secondary"
+                            startIcon={<AddCircleOutline />}
+                            onClick={() => {
+                                goTo(RouteName.CREATE_FORM, {
```
Comment (anagperal) [inline src/webapp/pages/event-tracker/EventTrackerPage.tsx:90]: Create a function with 
` goTo(RouteName.CREATE_FORM, { formType: "risk-assessment-summary" })`

File: src/webapp/pages/form-page/disease-outbreak-event/utils/updateDiseaseOutbreakEventFormState.ts [hunk 3]
```diff
@@ -9,22 +11,25 @@ import {
...
     const updatedFormWithRulesApplied = hasUpdatedFieldAnyRule
-        ? applyRulesInFormState(updatedForm, updatedField, diseaseOutbreakEventWithOptions.rules)
+        ? applyRulesInFormState(updatedForm, updatedField, configirableForm.rules)
         : updatedForm;
 
-    const fieldValidationErrors = validateForm(updatedFormWithRulesApplied, updatedField);
+    const fieldValidationErrors = validateFormState(
+        updatedFormWithRulesApplied,
+        updatedField,
+        configirableForm
```
Comment (anagperal) [inline src/webapp/pages/form-page/disease-outbreak-event/utils/updateDiseaseOutbreakEventFormState.ts:31]: configirableForm -->  configurableForm

File: src/data/repositories/OptionsD2Repository.ts (hunk 1 above)
Comment (9sneha-n) [reply src/data/repositories/OptionsD2Repository.ts:32]: will be taken care of in upcoming refactor PR

**Review by anagperal — APPROVED** [review]
thanks @9sneha-n !!

**Pull request merged**